domino_environments/_environments.py

```diff
@@ -67,6 +70,16 @@ def revision_download(self, environment_id, revision_id):
             + f"/v1/environments/{environment_id}/revisions/{revision_id}/dockerImageSourceProjectWeb"
         )
 
+    def revision_summaries(self, environment_id, pageStart, pageEnd):
+        return (
+            self.host
+            + f"/environments/{environment_id}/json/paged/{pageStart}/{pageEnd}"
+        )
+
+    def build_logs(self, build_logs_url: str):
+        return (
+            self.host + build_logs_url.replace("/logs","/fetchBuildLogsSince")
```

Comment:
Is there a better way to get the logs url? This doesn't seem to be useful if we are trying to access it programatically.

Reply:
It's the best way i've found so far. The replace is because the call to /logs gets redirected to fetchBuildLogsSince, but for some reason the requests library is not following the redirect, not sure why. We can replace this with a proper logs API if we find one, or one becomes available

Reply:
Ok no problem, looks good for now.